buildscripts/spotbugs-exclude.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <!-- Nullable-related stuff seem busted against this repo for whatever reason -->
+  <Match>
+    <Bug pattern="NP_NONNULL_PARAM_VIOLATION"/>
+  </Match>
+  <Match>
+    <Bug pattern="NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE"/>
+  </Match>
+  <Match>
+  <!-- This an an oddly specific one... -->
+    <Bug pattern="SC_START_IN_CTOR"/>
```

Comment:
can you add a comment about what this check is that we're ignoring? Maybe it's something we should fix?

Reply:
It's a check for "starting a thread in a constructor" - the cleanerThread in `WeakConcurrentMap`.  I'll adjust the comment.